webpack.config.js

```diff
@@ -26,7 +26,8 @@ module.exports = {
   },
   plugins: [
     new webpack.DefinePlugin({
-      SDK_VERSION: JSON.stringify(require('./package.json').version)
+      SDK_VERSION: JSON.stringify(require('./package.json').version),
+      STATE_TOKEN_COOKIE_NAME: JSON.stringify('oktaStateToken')
```

Comment:
And later on, if it's something that a consumer would want to override, adding the ability to override with a .authjsrc file (similar to how we do it in the okta-signin-widget)